tests/intfutil_test.py

```diff
@@ -160,14 +160,14 @@ def test_show_interfaces_description_etp9_in_alias_mode(self):
         assert result.exit_code == 0
         assert result.output == show_interface_description_eth9_output
 
-    def test_show_interfaces_description_etp33_in_alias_mode(self):
+    def test_show_interfaces_description_etp34_in_alias_mode(self):
```

Comment:
Once the 'internal interface' is removed for single ASIC these changes will not be required.

Reply:
yup